subprojects/packagefiles/sdl2/meson_options.txt

```diff
@@ -196,3 +196,9 @@ option(
     type : 'boolean',
     description : 'If possible, dlopen dependencies at runtime instead of linking (linux only)'
 )
+
+option(
+    'disable_sdl2main',
+    type : 'boolean',
+    description : 'Disable including the main function helper'
+)
```

Comment:
Maybe a `sdl2main` boolean option with a default value of `true` would be more in line with the Meson style recommendation of avoiding `enable-` (and by extension `disable-`) prefixed option names.

Reply:
I'd argue that this would overload the usage of `sdl2main` to mean an option and the library.

I think doing it the same way SDL does it upstream is preferable to reduce friction.

Reply:
I don't see how it's overloaded?

`sdl2main` is a meson.build variable, `-Dsdl2main` would be an option flag.

It's extremely disappointing that upstream has `sdl2.pc` but not `sdl2main.pc` but such is life...

Reply:
not in the sense of it overloading any variables, but overloading terminology.

but I'm in the minority here, so I shortened it to `sdl2main` and default to `true`